src/test/scala/tests/FailInAfterTest.scala

```diff
@@ -0,0 +1,21 @@
+package tests
+
+import co.helmethair.scalatest.helper.RegisterCall
+import org.scalatest.BeforeAndAfter
+import org.scalatest.funsuite.AnyFunSuite
+
+class FailInAfterTest extends AnyFunSuite with BeforeAndAfter with RegisterCall {
+  after {
```

Comment:
Style nitpick: empty line after class signature. You can also remove them, just be consistent!